src/controllers/trip.controller.js

```diff
@@ -0,0 +1,35 @@
+import TripService from '../services/trip.service';
+import AuthenticationService from '../services/authentication.service';
+import responseHandlers from '../utils/responseHandlers';
+import statusCodes from '../utils/statusCodes';
+import customMessages from '../utils/customMessages';
+import utils from '../utils/authentication.utils';
+
+const { updatedResponse, errorResponse } = responseHandlers;
+const { decodeToken } = utils;
+const { createReturnTrip } = TripService;
+const { findUserEmailIfExist } = AuthenticationService;
+
+/**
+   *@param {object} req request object
+   *@param {object} res response object
+   *@returns {object} response json object
+   *  @description User authentication class
+   */
+  export default class TripController {
+    /**
+   * @param {object} req request object
+   * @param {object} res response object
+   * @returns {object} response json object
+   * @description update password controller
+   */
+      static async returnTripCreation(req, res) {
+          try {
+        await createReturnTrip(req.body, req.tokenId); 
```

Comment:
could you create a variable call it `currentUserId` the you can pass `req.tokenId` like this `const currentUserId = req.tokenId`. the reason is because `req.tokenId` was kind of confusing to me at first, I had to trace its origin in order to understand what it contains. but if you use `currentUserId` I think it will be more understandable and clear.